mautic/mautic
# Rename apply button to save

For users it is confusing to have two buttons who do in parts (save) the same thing, but are named differently.
So the "apply" button should also be called "save"

![image](https://user-images.githubusercontent.com/13075514/114277883-6abe6300-9a2d-11eb-829a-ee58e3b3b058.png)


#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Create a new email. 
3. Check that the button is now labeled "save". Instead of "apply".
